blog/_posts/TeamDelSquared.md

```diff
@@ -0,0 +1,40 @@
+### Team DelSquared: Our Summer of Code, 2018
+
+ **And...it’s a wrap!**
```

Comment:
Hi @SharmisthaSG ! Thanks for the PR. But as you know our blog posts have particular format. And all posts need to follow that.

First of all, please add all the required parameters at the start of the file. You can check other blog posts or your own last blogpost to get an idea. :)